CHANGES.rst

```diff
@@ -3,6 +3,23 @@ Tequila-django
 
 Changes
 
+v 0.9.10+n on Month Day, Year
+--------------------------
+
+* Add additional env vars to account for Celery 4's renamed
+  configuration settings.
+
+  NOTE: if you are upgrading from Celery 3 to Celery 4, be
+  aware of the changes made to Celery's setting names.
+  Old setting names will still work after the change, but
+  users are encouraged to upgrade as soon as possible.
+  Settings can be upgraded automatically using the Celery 4
+  command line interface.
```

Comment:
An explicit statement of the affected vars (`CELERY_BROKER_HOST` and `CELERY_BROKER_PASSWORD`), and the affected files (your project's Django settings file(s) and celery.py) would be good, to give people the needed context before they have to dive into more verbose documentation.